korma/src/commonMain/kotlin/com/soywiz/korma/geom/Rectangle.kt

```diff
@@ -183,6 +186,13 @@ data class Rectangle(
         setTo(kotlin.math.ceil(x), kotlin.math.ceil(y), kotlin.math.ceil(width), kotlin.math.ceil(height))
         return this
     }
+
+    fun zeroSelf() {
+        x = 0.0
+        y = 0.0
+        width = 0.0
+        height = 0.0
+    }
```

Comment:
There is a `clear` function already that does this. But maybe a name like `setToZero` would fit better the naming convention here. What do you think? Maybe you can update the clear function to call setToZero and maybe mark clear as deprecated.

Reply:
Oups! My bad. Let's keep the `clear()` function. I'll replace it with `clear()` and get rid of `zeroSelf()`